test/specs/core/settle.spec.js

```diff
@@ -22,17 +22,41 @@ describe('core::settle', function() {
     expect(reject).not.toHaveBeenCalled();
   });
 
-  it('should resolve promise if validateStatus is not set', function() {
+  it('should resolve promise if validateStatus set null', function() {
     var response = {
       status: 500,
       config: {
+        validateStatus: null
       }
     };
     settle(resolve, reject, response);
     expect(resolve).toHaveBeenCalledWith(response);
     expect(reject).not.toHaveBeenCalled();
   });
 
+  it('should reject promise if validateStatus is not set', function() {
```

Comment:
this is the opposite of what we want, I believe. When someone doesn't set the `validateStatus`  config callback, the promise should resolve, similar to the previous functionality